packages/react-data-grid/src/Row.tsx

```diff
@@ -34,34 +33,44 @@ export default class Row<R> extends React.Component<IRowRendererProps<R>> implem
     e.preventDefault();
   };
 
-  getCell(column: CalculatedColumn<R>) {
-    const Renderer = this.props.cellRenderer!;
-    const { idx, cellMetaData, row, scrollLeft, isRowSelected, onRowSelectionChange } = this.props;
-    const { key } = column;
-
-    const cellProps: CellRendererProps<R> & { ref: (cell: CellRenderer | null) => void } = {
-      ref: (cell) => cell ? this.cells.set(key, cell) : this.cells.delete(key),
-      idx: column.idx,
-      rowIdx: idx,
-      height: this.props.height,
-      column,
+  getCells() {
+    const {
       cellMetaData,
-      value: this.getCellValue(key || String(column.idx) as keyof R) as R[keyof R], // FIXME: fix types
-      rowData: row,
-      expandableOptions: this.getExpandableOptions(key),
-      scrollLeft,
+      columns,
+      height,
+      idx,
       isRowSelected,
-      onRowSelectionChange
-    };
+      onRowSelectionChange,
+      row,
+      scrollLeft
+    } = this.props;
+    const Renderer = this.props.cellRenderer!;
+    const canSticky = isPositionStickySupported();
 
-    return <Renderer key={`${key as keyof R}-${idx}`} {...cellProps} />; // FIXME: fix key type
-  }
+    // FIXME: do we need this, considering columnsMetrics should have these columns sorted already?
+    const frozenColumns = columns.slice(0, this.props.lastFrozenColumnIndex + 1);
```

Comment:
I remember I tried in my local that to put `frozenColumns` first and then `nonFrozenColumns` later. The row cells displaying was screwed up. Not sure why that happened as I didn't take time to investigate.